codalab/apps/newsletter/models.py

```diff
@@ -0,0 +1,33 @@
+import hashlib
+import json
+
+import requests
+from django.db import models
+from django.db.models.signals import pre_delete
+from django.dispatch import receiver
+
+from codalab import settings
+
+
+class NewsletterUser(models.Model):
+    email = models.EmailField(null=True, blank=True, default=None)
```

Comment:
We probably don't want this to be nullable? We can't have a newsletter user without an email. 
I think we also want this to be `unique=True` so that we only create one object per email address.

Reply:
Updated in latest push. I had to make add some logic to the is_valid() function for the Unsubscribe form because the `unique=True` didn't like that we were passing non-unique emails to the unsub form.